### ISSUE TITLE
LDAP User Flags by Group doesn't allow providing a list for a flag

### ISSUE THREAD

**Issue by mprasil** [ansible/awx#703]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
I'd like to attach superuser role to two groups, AWX interface will allow saving such configuration, but it silently ignores the second group provided. 

##### ENVIRONMENT
* AWX version: 1.0.1.111
* AWX install method: docker on linux
* Ansible version:  2.4.0.0
* Operating System: 
* Web Browser: 

##### STEPS TO REPRODUCE

I'm setting is_superuser flag for two groups (Admins and Superadmins in example bellow)

```json
{
 "is_superuser": [
 	"CN=Admins,OU=Groups,DC=example,DC=com",
        "CN=Superadmins,OU=Groups,DC=example,DC=com"
 ]
}
```

##### EXPECTED RESULTS

Configuration is saved as provided, users from either group can log in and will be superusers.

##### ACTUAL RESULTS

When I provide such configuration in the admin interface, AWX will accept the configuration, but when visiting the settings again, it will show configuration reduced to a version, that's using just the first group:

```json
{
 "is_superuser": "CN=Admins,OU=Groups,DC=example,DC=com"
}
```

Also only the "Admins" group users will get superuser rights upon logging in.

##### ADDITIONAL INFORMATION

This can be obviously worked around by creating a group in LDAP, that will contain both groups, but sometimes that might be impractical from organization perspective. At least, AWX should warn about such configuration and fail to save it.

**Comment by wenottingham** [comment]
Marking as a bug... since this isn't allowed, it should error on save.

**Comment by cchurch** [comment]
This format *is* supported by [django-auth-ldap](https://bitbucket.org/psagers/django-auth-ldap/src/ec27057388fb5c55013d5d8ec43e995897e35f97/django_auth_ldap/backend.py?at=1.2.x&fileviewer=file-view-default#backend.py-628), but not by the current settings validation. It should be easy to add though...

**Comment by megakoresh** [comment]
This would be extremely useful feature to have, and it also improves consistency with the other fields.

**Comment by ryanpetrello** [comment]
We've just released 1.0.7, which provides support for this feature. You can try it out here: https://github.com/ansible/awx/releases/tag/1.0.7

Let us know if you're still having problems after installing the latest awx - thanks!
